README.md

```diff
@@ -30,21 +30,23 @@ General guidelines for Hellbender developers
 
 * Hellbender is  BSD licensed.  The license is in the top level LICENSE.TXT file.  Do not add any additional license text or accept files with a license included in them.
 
-* Untested code is considered **non-existent** and thus is subject to removal at any time (exception is main methods, or super corner conditions, or `toString()` code). New code without tests should not be accepted as pull requests.
+* Untested code is considered **non-existent** and thus is subject to removal at any time (exception is main methods, or super corner conditions, or `toString()` code).
 
-* Do use Java 8 features liberally, in particular streams, functional programming etc.
+* Don't accept pull requests that introduce warnings. Warnings need to be addressed or supressed.
```

Comment:
We should add this check to travis if we want to enforce it.  Adding a ticket for this #368.